src/mumble/GlobalShortcut.cpp

```diff
@@ -625,6 +625,27 @@ void GlobalShortcutConfig::on_qpbRemove_clicked(bool) {
 	qlShortcuts.removeAt(idx);
 }
 
+void GlobalShortcutConfig::on_qpbImport_clicked() {
+	commit();
+	QString filePath = QFileDialog::getOpenFileName(this, tr("Import shortcuts"), QString(), QLatin1String("*.ini"));
+	if (filePath.isEmpty()) {
+		return;
+	}
+	QSettings s(filePath, QSettings::IniFormat);
+	qlShortcuts = g.s.loadShortcuts(&s);
```

Comment:
Won't doing it this way also nuke the server specific shortcuts?

Reply:
I don’t remember…
Export says it ignores server specific shortcuts, and that just iterates only `qlShortcuts`.

Reply:
Yes it does. Shortcuts are added externally into `qlShortcuts`.